core/tests/webdriverio_utils/StoryEditorPage.js

```diff
@@ -86,18 +91,25 @@ var StoryEditorPage = function() {
   var chapterTitlesSelector = function() {
     return $$('.e2e-test-chapter-title');
   };
+  var chapterStatusSelector = function() {
+    return $$('.e2e-test-chapter-status');
+  };
   var discardOption = $('.e2e-test-show-discard-option');
   var discardChangesButton = $('.e2e-test-discard-story-changes');
   var explorationAlreadyPresentMsg = $('.e2e-test-invalid-exp-id');
   var explorationIdInput = $('.e2e-test-exploration-id-input');
   var explorationIdSaveButton = $('.e2e-test-exploration-id-save-button');
   var nextChapterCard = $('.e2e-test-next-chapter-card');
   var nodeDescriptionInputField = $('.e2e-test-add-chapter-description');
+  var plannedPublicationDateInput = $(
+    '.e2e-test-planned-publication-date-input');
   var nodeOutlineEditor = $('.e2e-test-add-chapter-outline');
   var nodeOutlineFinalizeCheckbox = $('.e2e-test-finalize-outline');
   var nodeOutlineEditorRteContentSelector = function() {
     return $$('.e2e-test-rte');
   };
+  var publishUptoChaptersDropdownSelector = $(
```

Comment:
This `Selector` keyword creates confusion with other element selector methods having `selector` suffixes. I think `Selector` suffix for the selector element is fine, we should do something different for element selector methods. I'll think about it and update this thread!